src/commands/balances/index/processor.ts

```diff
@@ -1123,3 +1152,29 @@ export async function getBalanceTokens(i: ButtonInteraction) {
 
   return availableTokens
 }
+
+export async function unlinkWallet(
+  msg: OriginalMessage,
+  author: User,
+  addressOrAlias: string
+) {
+  const profileId = await getProfileIdByDiscord(author.id)
+  const { ok, status, log, curl } = await profile.disconnectOnChainWallet(
+    profileId,
+    addressOrAlias
+  )
+  // wallet not found
+  if (!ok && status === 404) {
+    throw new InternalError({
+      msgOrInteraction: msg,
+      title: " Invalid wallet information",
+      description: `Your address or alias is uncorrected`,
```

Comment:
```suggestion
      descriptions: ["We couldn't process that request"],
      reason: "Address or alias is incorrect"
```
chuyển qua dùng format mới nha em